docs/guides/getting-started/installing-cypress.mdx

```diff
@@ -78,6 +78,35 @@ yarn add cypress --dev
 System proxy properties `http_proxy`, `https_proxy` and `no_proxy` are respected
 for the download of the Cypress binary.
 
+### <Icon name="terminal" /> `pnpm install`
+
+Installing Cypress via [`pnpm`](https://pnpm.io/):
+
```

Comment:
Let's simplify this to not use an actual example  and to be more generic and in line with the `npm` and `yarn` installs examples above

Maybe something like:


### <Icon name="terminal" /> `pnpm add`

Installing Cypress via [`pnpm`](https://pnpm.io):

```shell
cd /your/project/path
```
```shell
pnpm add cypress -D
```

:::info

You need to make sure that you have the `pnpm` environment installed locally.
If not you need to install it: `npm install pnpm@latest -g`.

:::

Reply:
If the bit about `npm install pnpm@latest -g` is added here, then it should also be added for yarn i.e. `npm install yarn@latest -g`, since Node.js only comes with npm installed. GitHub runners are a bit different, they have npm and yarn pre-installed but not pnpm.

Reply:
🫤， Oh My use of pnpm was a really elementary mistake！